webpack.config.js

```diff
@@ -36,6 +36,15 @@ module.exports = {
                     'css-loader',
                     'postcss-loader'
                 ]
+            },
+            {
+                test: /\.(png|jpe?g|gif)$/,
```

Comment:
Let's not use a file loader. Let's use express js assets. So save this image on the server (preferrably in the public folder) and load it as an express asset.

Reply:
I tried loading it as an express asset from ./server/public/images but kept getting this error message
![image](https://user-images.githubusercontent.com/35481645/57969432-ba45a280-796e-11e9-87e9-3b206fc0acc9.png)
I have tried solving this issue with @devPinheiro but he seems not to also understand what the problem is ...I would be returning to file loader now pending the time you see this

Reply:
Problem resolved. thank you